decoder.rb

```diff
@@ -0,0 +1,44 @@
+morse_code = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def decode_char(code)
+  morse_code[code]
+end
```

Comment:
- [ ] Initially, when I ran your code, it throws an error that `morse_code` is not defined local variable as you can see below but after I made a correction to your code and makes the morse_code a constant variable, your code works with no error so I think you should make the morse code a constant variable. :blush: 
  
  ![image](https://github.com/codedit334/Morse_Decoder/assets/92303891/77097d48-652d-40a1-8eac-fd9998ab35a8)